src/latis-python-client/latis.py

```diff
@@ -46,14 +79,39 @@ def data(baseUrl, dataset, returnType,
 
 
 def download(baseUrl, dataset, filename, fileFormat,
-             projections, selections, operations, preferVersion2=False):
+             projections=[], selections=[], operations=[],
+             preferVersion2=False):
+    """Download dataset from Latis instance to file.
+
+    Args:
+        baseUrl (str): The base URL of the Latis instance.
+        dataset (str): The name of the dataset.
+        filename (str): The name of the file to download.
+        fileFormat (str): The format of the file to download.
+        projections (list, optional): List of projections to use.
+        selections (list, optional): List of selections to use.
+        operations (list, optional): List of operations to apply.
+        preferVersion2 (bool, optional): If True, prefer Latis version 2.
+    """
+
     latis3 = __datasetWillUseVersion3(preferVersion2)
```

Comment:
"In the `download()` function, you are missing the argument `baseUrl` in the call to `__datasetWillUseVersion3()`. This isn't an issue with the docstring per se, but a potential bug in your code."